modules/local/whatshap/stats/main.nf

```diff
@@ -23,7 +23,7 @@ process WHATSHAP_STATS {
     whatshap stats \\
         $args \\
         --sample ${meta.id} \\
-        --tsv ${prefix}.stats.tsv.gz \\
+        --tsv ${prefix}.stats.tsv \\
```

Comment:
Is this one of those tools that deduces output format from the file extension? You don't need to inform the tool using a parameter?

Reply:
Ah okay! :D